scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -428,3 +354,46 @@ task GetSampleIds {
     File gvs_ids = "gvs_ids"
   }
 }
+
+task CurateInputLists {
+  input {
+    String dataset_name
+    String project_id
+    File input_vcf_index_list
+    File input_vcf_list
+    File input_sample_map
+    File input_sample_name_list
```

Comment:
Might want to rename one of these parameters (input_sample_name_map or input_sample_name_list) to make it clear that they don't represent the same thing. e.g. input_sample_map => input_samples_to_be_loaded_map?